docs/spec/spec.md

```diff
@@ -0,0 +1,379 @@
+# Specification: Ballerina WebSubHub Library
+
+_Owners_: @shafreenAnfar @chamil321 @ayeshLK    
+_Reviewers_: @shafreenAnfar    
+_Created_: 2021/11/23  
+_Updated_: 2021/11/23  
+_Issue_: [#2162](https://github.com/ballerina-platform/ballerina-standard-library/issues/2162)
+
+# Introduction
+
+This is the specification for WebSubHub standard library, an API framework to implement WebSub compliant `hub` services 
+and `publisher` clients using [Ballerina programming language](https://ballerina.io/).  
```

Comment:
For the sake of consistency, lets stick to this `Introduction`

https://github.com/ballerina-platform/module-ballerina-jwt/blob/master/docs/spec/spec.md